image/rhel/konflux.Dockerfile

```diff
@@ -40,10 +40,11 @@ COPY . .
 RUN git restore scripts/konflux/bootstrap-yarn/package-lock.json && \
     scripts/konflux/fail-build-if-git-is-dirty.sh
 
+ARG VERSIONS_SUFFIX
+ENV MAIN_TAG_SUFFIX="$VERSIONS_SUFFIX" COLLECTOR_TAG_SUFFIX="$VERSIONS_SUFFIX" SCANNER_TAG_SUFFIX="$VERSIONS_SUFFIX"
+
 ENV GOFLAGS=""
 ENV CGO_ENABLED=1
-# TODO(ROX-19958): figure out if we need BUILD_TAG
```

Comment:
Since you removed this comment, is it "figured out"?

Reply:
For me - yes. Please ask a question if you'd like me to clarify anything.

Reply:
Yes, how does the image know its own version? 

![Screenshot 2024-05-29 at 14 05 20](https://github.com/stackrox/stackrox/assets/9576848/ccd8a51e-e1e0-4229-9625-a06922185da3)

Reply:
Clarified on a call that this version comes from `status.sh`.